app/models/question.rb

```diff
@@ -3,6 +3,7 @@
 class Question < ApplicationRecord
   has_and_belongs_to_many :answer_possibilities
   has_many :answers, dependent: :destroy
+  has_one :survey, through: :question_group, inverse_of: :questions
```

Comment:
has_one? Can't a question be added to multiple surveys?

Reply:
Currently not as you would need a joining table. But I have in mind that we discussed that we don‘t want that as you would rather provide a survey duplication feature